operator/hack/common.sh

```diff
@@ -39,21 +39,15 @@ function apply_operator_manifests() {
       disable_security_context_config=""
   fi
 
-  if [[ "${USE_MIDSTREAM_IMAGES}" == "true" ]]; then
-    # Get Operator channel from json for midstream
-    operator_channel=$(< midstream/iib.json jq -r '.operator.channel')
   env -i PATH="${PATH}" \
-    INDEX_VERSION="${index_version}" OPERATOR_VERSION="${operator_version}" NAMESPACE="${operator_ns}" OPERATOR_CHANNEL="${operator_channel}" \
-    IMAGE_TAG_BASE="${image_tag_base}" DISABLE_SECURITY_CONTEXT_CONFIG="${disable_security_context_config}" \
-    envsubst < "${ROOT_DIR}/operator/hack/operator-midstream.envsubst.yaml" \
-    | "${ROOT_DIR}/operator/hack/retry-kubectl.sh" -n "${operator_ns}" apply -f -
-  else
-  env -i PATH="${PATH}" \
-    INDEX_VERSION="${index_version}" OPERATOR_VERSION="${operator_version}" NAMESPACE="${operator_ns}" \
-    IMAGE_TAG_BASE="${image_tag_base}" DISABLE_SECURITY_CONTEXT_CONFIG="${disable_security_context_config}" \
+    INDEX_IMAGE_TAG="${index_image_tag}" \
+    STARTING_CSV="rhacs-operator.${starting_csv_version}" \
+    NAMESPACE="${operator_ns}" \
+    OPERATOR_CHANNEL="${operator_channel}" \
+    INDEX_IMAGE_REPO="${index_image_repo}" \
```

Comment:
Minor and optics: let's please put `INDEX_IMAGE_REPO` before `INDEX_IMAGE_TAG`. How about this reordering?
```suggestion
    NAMESPACE="${operator_ns}" \
    INDEX_IMAGE_REPO="${index_image_repo}" \
    INDEX_IMAGE_TAG="${index_image_tag}" \
    STARTING_CSV="rhacs-operator.${starting_csv_version}" \
    OPERATOR_CHANNEL="${operator_channel}" \
```

This way values follow the order of `operator/hack/olm-operator-install.sh` arguments. Not strictly necessary but makes it nicer.